src/services/insight.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import { BITCOIN_INSIGHT_URL } from 'constants/bitcoinConstants';
+
+const successfulResponse = (response): boolean => {
+  return response.ok && response.status >= 200 && response.status < 300;
```

Comment:
Any explanations under `response.status >= 200 && response.status < 300`? Because 204 can be and error while 301 can be content result that has been redirected.

Reply:
As well `response.ok` is the same as the `response.status >= 200 && response.status < 300`, I think it's enough to have `response.ok && response.status === 200` here.

Ref - https://developer.mozilla.org/en-US/docs/Web/API/Response/ok.

Reply:
You're probably right, will check that. 204 should not represent an error, BTW.